HEC.FDA.Model/stageDamage/ImpactAreaStageDamage.cs

```diff
@@ -637,7 +635,7 @@ public void ReportProgress(object sender, ProgressReportEventArgs e)
         internal List<string> ProduceImpactAreaStructureDetails(Dictionary<int, string> impactAreaNames)
         {
             //this list will be the size of the number of structures + 1 where the first string is the header
-            List<DeterministicOccupancyType> deterministicOccupancyTypes = Inventory.SampleOccupancyTypes(new compute.MedianRandomProvider(), computeIsDeterministic: true);
+            List<DeterministicOccupancyType> deterministicOccupancyTypes = Inventory.SampleOccupancyTypes(iteration:1, computeIsDeterministic: true);
```

Comment:
Looks like we previously wanted the mean here. now we're getting the iteration 1 sample. Are we still good?

Reply:
Add comments to note that iteration is ignored for compute is deterministic

Reply:
I have added public XML comments on all of the objects that can be sampled